icu4c/source/configure

```diff
@@ -9127,16 +9115,22 @@ if test -z "$PYTHON";
 then
     echo "" > data/rules.mk
 else
-    echo "Spawning Python to generate data/rules.mk..."
-    PYTHONPATH="$srcdir/data" $PYTHON -m buildtool \
-        --mode gnumake \
-        --seqmode parallel \
-        --src_dir "$srcdir/data" \
-        --filter_file "$ICU_DATA_FILTER_FILE" \
-        $BUILDTOOL_OPTS \
-        > data/rules.mk
-    if test "$?" != "0"; then
-        as_fn_error $? "Python failed to run; see above error." "$LINENO" 5
+    if test -f "$srcdir/data/locales/resfiles.mk";
```

Comment:
This works for ICU 64 but isn't future-proof, because resfiles.mk might be removed in the future.  Can you check for a different file?  For example, data/misc/supplementalData.txt?

Reply:
Another good file: data/locales/root.txt

Reply:
@sffc probably better would be to put the check into python itself. but, i will change it to `data/locales/root.txt`

Reply:
I agree about checking in Python, but the other objective is to avoid the need for Python being present in the tarball build, so it is principled to do a check in configure.